ofrak_core/Dockerstub

```diff
@@ -61,8 +61,11 @@ RUN cd /tmp && \
 
 # Install Jefferson
 WORKDIR /tmp
-RUN wget https://bootstrap.pypa.io/pip/get-pip.py && python3.9 get-pip.py && python3.8 get-pip.py && rm get-pip.py
-RUN python3.9 -m pip install jefferson
+RUN wget https://bootstrap.pypa.io/pip/get-pip.py && \
+    /usr/bin/python3 get-pip.py && \
+    /usr/local/bin/python get-pip.py && \
+    rm get-pip.py
+RUN /usr/bin/python3 -m pip install jefferson
```

Comment:
Investigating this, looks like this is an artifact of #326 which added Jefferson back when the Docker image was based on python 3.7, which Jefferson didn't support. That pull request later bumped the Python version to 3.8 and added jefferson to our requirements.txt, so we end up having the `/usr/local/bin/python` (docker image) install of jefferson instead of the `/usr/bin/python` (debian) jefferson on our `PATH` anyway.

Also, `pip` always on the python version we actually use.

tl;dr this whole section is not needed.